modules/oadp-creating-restore-cr.adoc

```diff
@@ -55,3 +55,10 @@ $ oc get restore -n openshift-adp <restore> -o jsonpath='{.status.phase}'
 $ oc get all -n <namespace> <1>
 ----
 <1> Namespace that you backed up.
+
+. If you use Restic to restore DeploymentConfigs or if you use post-restore hooks, run a cleanup script by entering the following command:
```

Comment:
I'm thinking we may want to add a quick explanation as to why this is needed. "To prevent the cluster from deleting the restored DC pods immediately on restore, the OADP velero plugins scale down the DeploymentConfig and restore the pods as standalone pods to allow restic and post-restore hooks to complete their actions on the restored pod. The cleanup script will remove these disconnected pods and scale any DeploymentConfigs back up to the appropriate number of replicas."

Feel free to reword the above as needed.

Reply:
@sseago should we link this explanation to the incremental restore too somehow? so it would be clear it one should also execute it before incremental restore?

Reply:
@sbeskin-redhat Yes, the change looks good.
@mperetzred So this relates to *any* restore where we're restoring DeploymentConfigs and their pods, incremental or otherwise. But wouldn't the whole "creating a restore CR" section be relevant in the incremental case?